rbi/yast/i18n.rbi

```diff
@@ -0,0 +1,15 @@
+# typed: strong
+
+# Namespacing note:
+# if we say `module Yast::I18n` then String means ::String
+# if we said `module Yast; module I18n` then String would mean Yast::String
```

Comment:
maybe it would be better to be explicit and just use ::String?

Reply:
Yast::String should be destroyed, it is a design bug in YCP Killer.
Do you really want to write `::String` instead of `String` everywhere?
Fortunately the two have such different APIs that an accidental confusion is resolved pretty quickly.